batch7crud-roll4/src/main/java/com/lftechnology/batch7crud/controller/EmployeeController.java

```diff
@@ -0,0 +1,215 @@
+                    request.setAttribute("message", e.getMessage());
+
+                    RequestDispatcher view = request.getRequestDispatcher("/WEB-INF/views/error.jsp");
+                    view.forward(request, response);
+                }
+            } else if (paths[2].equals("delete")) {
+                try {
+                    int id = Integer.parseInt(paths[1]);
+                    deleteProcess(request, response, id);
+                } catch (NumberFormatException e) {
+                    e.printStackTrace();
+                    response.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
+                    request.setAttribute("message", e.getMessage());
+
+                    RequestDispatcher view = request.getRequestDispatcher("/WEB-INF/views/error.jsp");
+                    view.forward(request, response);
+                }
+
+            }
+        }
+    }
+
+    private void list(HttpServletRequest request, HttpServletResponse response, int page) throws ServletException, IOException {
+        try {
+            String forward = "/WEB-INF/views/employees.jsp";
+
+            int noOfRecords = employeeService.fetchNoOfRecords();
+            int noOfPages = (int) Math.ceil(noOfRecords * 1.0 / 20);
+            int recordLimit = 20;
+
+            request.setAttribute("employees", employeeService.fetch(page, recordLimit));
+            request.setAttribute("currentPage", page);
+            request.setAttribute("noOfPages", noOfPages);
+
+            RequestDispatcher view = request.getRequestDispatcher(forward);
+            view.forward(request, response);
+
+        } catch (DataException e) {
+            e.printStackTrace();
+            response.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
+            request.setAttribute("message", e.getMessage());
+
+            RequestDispatcher view = request.getRequestDispatcher("/WEB-INF/views/error.jsp");
+            view.forward(request, response);
+        }
+    }
+
+    private void create(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        String forward = "/WEB-INF/views/create.jsp";
+
+        RequestDispatcher view = request.getRequestDispatcher(forward);
+        view.forward(request, response);
+    }
+
+    private void createProcess(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        String firstName = request.getParameter("firstName");
+        String lastName = request.getParameter("lastName");
+        String station = request.getParameter("station");
+
+        Employee employee = new Employee();
+        employee.setFirstName(firstName);
+        employee.setLastName(lastName);
+        employee.setStation(station);
+
+        try {
+            employeeService.save(employee);
+            response.sendRedirect(request.getContextPath() + "/employees");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "/employees" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)